pinot-core/src/main/java/org/apache/pinot/core/query/reduce/HavingFilterHandler.java

```diff
@@ -30,7 +30,12 @@ public class HavingFilterHandler {
   private final RowMatcher _rowMatcher;
 
   public HavingFilterHandler(FilterContext havingFilter, PostAggregationHandler postAggregationHandler) {
```

Comment:
Suggest removing this one if it is just used by tests. We want to enforce people to support null handling from now on

Reply:
@Jackie-Jiang I am sorry I don't follow. What do you mean by removing this one?

Reply:
What I meant is to remove this constructor, and always use the one with `nullHandlingEnabled` passed in